packages/pluggableWidgets/heatmap-chart-web/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "heatmap-chart-web",
+  "widgetName": "HeatMap",
+  "version": "1.0.0",
```

Comment:
Version is incorrect as HeatMap was already published in the past. It should be 3.0.0
https://github.com/mendix/charts/blob/master/src/package.xml